docs/upgrade/upgrade-opensource/upgrade-guide-from-5.2-to-5.4/upgrade-guide-from-5.2-to-5.4-generic.rst

```diff
@@ -117,6 +117,21 @@ node before validating that the node you upgraded is up and running the new vers
   or running repairs.
 * Not to apply schema changes.
 
+**After** the upgrade, you need to verify that Raft was successfully initiated in 
+your cluster. You can skip this step only in the following cases:
+
+* The ``consistent_cluster_management`` option was enabled in a previous 
+  ScyllaDB version.
+* You disabled the ``consistent_cluster_management`` option before upgrading 
```

Comment:
Can we make it more clear that this is a logical disjunction? (e.g. "Or you disabled...") Otherwise one may misinterpret that both bullets need to be true.

Reply:
Hm, it starts with "only in the following cases:"
maybe it's OK.

Although how about
"only in either of the following cases:"
?